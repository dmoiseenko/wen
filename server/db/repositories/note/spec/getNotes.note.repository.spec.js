const { getNotes } = require('../getNotes.note.repository');
const bootstrap = require('../../../bootstrap/bootstrap.js');


beforeEach(() => bootstrap());

it('should get all notes', async () => {
  const actual = await getNotes();

  expect(actual).toMatchSnapshot();
});
