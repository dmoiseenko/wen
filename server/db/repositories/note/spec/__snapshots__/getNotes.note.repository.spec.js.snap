// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get all notes 1`] = `
Array [
  Object {
    "id": 1,
    "text": "asd1",
  },
  Object {
    "id": 2,
    "text": "asd2",
  },
  Object {
    "id": 3,
    "text": "asd3",
  },
]
`;
